Library/Formula/ice.rb

```diff
@@ -13,26 +10,19 @@ class Ice < Formula
     sha1 "7bdb347052b8df900c2c2e6797726a79d9c0242d" => :mountain_lion
   end
 
-  option 'doc', 'Install documentation'
-  option 'demo', 'Build demos'
+  option "without-java", "Compile without Java support."
+
+  depends_on :java => "1.7" if build.with? "java"
 
```

Comment:
Nope. `=>` in this context means "This version or any greater version".

If you look at our dependency handling you can see `MinimumMacOSRequirement` resolves to `:macos` [here](https://github.com/Homebrew/homebrew/blob/master/Library/Homebrew/dependency_collector.rb#L99). There is no "pure" usage of `MinimumMacOSRequirement` in any core formula.